### PR TITLE
[FLINK-34187] Setup CI for Flink CDC project

### PR DESCRIPTION
**Pull request by GOODBOY008**
Migrate ci from aws to github action

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@leonardBang  Please take a look , and I will polish after confirm the goal is right.

**Review by leonardBang — APPROVED** [review]
Cool, Thanks @GOODBOY008 for the contribution

File: .github/workflows/flink_cdc.yml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+              sudo /tmp/jattach $java_pid jcmd "Thread.print -l"
+              sudo /tmp/jattach $java_pid jcmd GC.heap_info || true
+            done
+          else
+            for java_pid in $(jps -q -J-XX:+PerfDisableSharedMem); do
+              echo "----------------------- pid $java_pid -----------------------"
+              jcmd $java_pid VM.command_line || true
+              jcmd $java_pid Thread.print -l
+              jcmd $java_pid GC.heap_info || true
+            done
```
Comment (leonardBang) [inline .github/workflows/flink_cdc.yml:215]: I'm not familiar with this, does flink repo has similar tools?

Comment (GOODBOY008) [reply .github/workflows/flink_cdc.yml:215]: @leonardBang I copy this from https://github.com/apache/flink-connector-shared-utils/blob/b5ad097df25973ad05d0d6af0f988b65b0d8cd22/.github/workflows/ci.yml#L183. Which is connectors ci parent.

Comment (leonardBang) [reply .github/workflows/flink_cdc.yml:215]: Thanks for your confirm, makes sense to me

**Pull request merged**